app/models/cait_indc/submission.rb

```diff
@@ -0,0 +1,16 @@
+module CaitIndc
+  class Submission < ApplicationRecord
+    belongs_to :location
+    validates :submission_type, presence: true
+    validates :language, presence: true
+    validates :submission_date, presence: true
+    validates :url,
+              presence: true,
+              format: URI.regexp(%w(http https))
+
+    def submission_date=(val)
+      parts = val.split('/')
```

Comment:
Add format validation to ensure that the data is passed as mm/dd/yyyy with `/` as the separator.